fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/explain/NodeInfo.java

```diff
@@ -154,6 +154,12 @@ public class NodeInfo {
             NodeIcon.COMPUTATION_OPERATOR,
             "Unordered Distinct",
             "An unordered distinct operator processes its input records and returns the set of distinct records. It does not require its input records to be ordered but does impose a memory overhead as it needs to track records it has already encountered.");
+
+    public static final NodeInfo UNORDERED_UNIQUE_OPERATOR = new NodeInfo(
```

Comment:
I don't think there is an operator like that planned. Can't we just remove this?

Reply:
done.